docs/function/handlers.md

```diff
@@ -224,6 +224,22 @@ Note that path parameters (e.g. `{id}` in the example above) are available as re
 $id = $request->getAttribute('id');
 ```
 
+### Event and context
+
+The HTTP request event and context are available as attributes on the request:
+```php
+/** @var $event Bref\Event\Http\HttpRequestEvent */
+$event = $request->getAttribute('lambda-event'); 
+
+/** @var $context Bref\Context\Context */
+$context = $request->getAttribute('lambda-context');
+```
+
+If you're looking for the request context array, for example when using a [Lambda authorizer](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-lambda-authorizer.html#http-api-lambda-authorizer.payload-format-response):
+```php
+$requestContext = $request->getAttribute('lambda-event')->getRequestContext(); 
+```
+
 [Full reference of HTTP events in `serverless.yml`](https://www.serverless.com/framework/docs/providers/aws/events/http-api/).
```

Comment:
Can this line be moved above the new section? It seems to me like it belongs to what's above, not to "Lambda event and context"